llvm/lib/IR/Core.cpp

```diff
@@ -3555,11 +3550,6 @@ LLVMValueRef LLVMBuildNSWNeg(LLVMBuilderRef B, LLVMValueRef V,
   return wrap(unwrap(B)->CreateNSWNeg(unwrap(V), Name));
 }
 
-LLVMValueRef LLVMBuildNUWNeg(LLVMBuilderRef B, LLVMValueRef V,
-                             const char *Name) {
-  return wrap(unwrap(B)->CreateNUWNeg(unwrap(V), Name));
-}
```

Comment:
We should avoid dropping C APIs if it's still possible to support them (e.g. by explicitly setting the flag, or even just creating the instruction without the flag).

Reply:
I will make them deprecated.

Reply:
Done.